fossevents/pages/templates/pages/event_detail.html

```diff
@@ -1,22 +1,33 @@
 {% extends "base.html" %}
 {% load django_markdown %}
-{% block body_classes %}{{block.super}}page--event-detail{% endblock body_classes %}
-{% block title %}{{ event.name }} • {{block.super}}{% endblock title %}
 
 {% block content %}
 
-<div class="eventDetail" role='main'>
+<div class="col-md-offset-3 col-md-6">
+            <div class="inner">
+               <div class="panel panel-default">
+                  <div class="panel-heading">
+                     <div class="info">
+                        <h2 class="title">{{ event.name }}</h2>
+                        <span class="desc">{{ event.start_date }}</span>
+                        <p class="desc">Address - 23,Nimhans,Bangalore-190001</p>
+                     </div>
+                     <div class="info">
+                        <span >Contact Person:Saurabh Kumar</span>
```

Comment:
We don't want any hard coded values in templates. Please only use template variables.